manifests/m/Microsoft/SQLServerManagementStudio/Preview/21.0/Microsoft.SQLServerManagementStudio.21.Preview.yaml

```diff
@@ -0,0 +1,6 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
```

Comment:
Please update file name and path to match this change.

Reply:
I removed the (redundant and problematic) "21" from a few places... let's see how it foes this time around.